spec/services/users/profile_image_generator_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
```

Comment:
If `Users::ProfileImageGenerator` is kept as a module, then this spec will need this setup to mimic when other class might utilize it. It works but I'm uncertain if this will break _other_ test setups.
```suggestion
require "rails_helper"

RSpec.configure do |c|
  include CloudinaryHelper
end

```